qdev_wrappers/station_configurator.py

```diff
@@ -155,14 +147,12 @@ def setup_parameter_from_dict(p, options):
 
         # add the instrument to the station
         self.station.add_component(instr)
+
+        # restart the monitor to apply the changes
```

Comment:
Is this restart really needed anymore? Creating a new monitor should do the job?